helm/index.yaml

```diff
@@ -16,50 +44,50 @@ entries:
     digest: a39ab0af4e8b3c179516b071a79232d52d987b547cfb358bd46140dd27ce2530
     home: https://pinot.apache.org/
     keywords:
-      - olap
-      - analytics
-      - database
-      - pinot
+    - olap
+    - analytics
+    - database
+    - pinot
     maintainers:
-      - email: dev@pinot.apache.org
-        name: pinot-dev
+    - email: dev@pinot.apache.org
+      name: pinot-dev
     name: pinot
     sources:
-      - https://github.com/apache/pinot/tree/master/helm
+    - https://github.com/apache/pinot/tree/master/helm
     urls:
-      - pinot-0.2.9.tgz
```

Comment:
Please only modify the 0.3.0 section, don't change the older ones.

Reply:
Gotcha, sorry about that. Should be fixed now except a trailing EOL on the `index.yaml` file. Will this be ok?